packages/react-component-library/src/components/Modal/Footer.tsx

```diff
@@ -13,7 +13,14 @@ export const Footer: React.FC<FooterProps> = ({
   tertiaryButton,
 }) => {
   return (
-    <footer className="rn-modal__footer" data-testid="modal-footer">
+    <footer
+      className={
+        tertiaryButton
```

Comment:
We try to use the classNames utility instead of ternaries. This could become:

```javascript
import classNames from 'classnames'

// ...

const classes = classNames('rn-modal__footer', {
  'rn-modal__footer--has-tertiary': tertiaryButton
})

<footer className={classes} ...
```

Reply:
Thought I may had strayed from convention there...